### ISSUE TITLE
3.0.10 broke DS4W, does not see the controller.

### ISSUE THREAD

**Issue by MFS2020** [Jays2Kings/DS4Windows#691]
I was really hesitant updating seeing so many changes, and my gut was right, immediately after updating - the controller is no longer seen by the DS4Windows, during the update process 

I had an error saying some process was interfering with update completion, I did not get a chance to continue installation after terminating the process (it was not ds4windows process, some other service). 

Then I relaunched the updated and after successfully updating -- I could no longer use the controller as ds4windows didn't see it.

I tried reinstalling and using older versions.
I did not have any windows updates.

Update: The issue was solved by deleting inputmapper, this prompted the DS4Windows to reinstall the initial installation of drivers prompt which fixed the controllers, would be nice to have a button that relaunches that window that installs drivers like hiddeninput etc., that is what solved the issue., for now. Also I am back on version 3.0.8 so not sure if 10 will work, ill wait till next version comes out to update.

**Comment by AnyHoLiC-F** [comment]
Same issue here. Deleted everything and went back to v3.0.9 which seems to be stable.

Also, I just noticed the program doesn't have a .xml file for settings/user preferences. It would be a nice a feature especially when reinstalling the program in cases like this.

**Comment by MFS2020** [comment]
the xml file with settings and profiles is stored in Application Data folder, so you can delete and reinstall and it should still all be saved there, that's the case for me at least..

**Comment by AnyHoLiC-F** [comment]
> the xml file with settings and profiles is stored in Application Data folder, so you can delete and reinstall and it should still all be saved there, that's the case for me at least..

Yeah. But ever since I reinstalled Windows 10, I made "Program Folder" my choice for easy access to profiles. I just found it, it's called "Profiles.xml" It's in the main directory of the program, DS4Windows folder.

One other thing: **Why is this issue opened under "Jays2Kings"? What happened here?!**
https://github.com/Ryochan7/DS4Windows/issues/   seems like it doesn't exist. It seems that _someone disabled the "issues" for their repository._ 👀
